src/Core/Components/CounterBadge/FluentCounterBadge.razor.css

```diff
@@ -20,6 +21,12 @@
     cursor: default;
 }
 
-    .counterbadge:empty {
+    .counterbadge:not(.dot):empty {
         display: none;
     }
+
+.dot {
```

Comment:
We shouldn't use "generic / basic" class names, to avoid conflicts with other existing libraries.
In this case, `dot` is probably already in use.
We should use a `fluentui` prefix for all classes.

In this case, it would be preferable to use a `dot-only` attribute (for example):
```xml
<div class="@ClassValue" style="@StyleValue" dot-only></div>
```

Same for `counterbadge` :-)

Reply:
Good catch. Obviously code from the early days. Will make it work with attribute